src/home_robot_hw/home_robot_hw/env/visualizer.py

```diff
@@ -235,6 +236,22 @@ def visualize(
             color = map_color_palette[9:12][::-1]
             cv2.drawContours(self.image_vis, [agent_arrow], 0, color, -1)
 
+        elif rl_obs_frame is not None:
```

Comment:
What is RL obs frame supposed to be? Can we update docstring?

Reply:
It is supposed to show the observations that go as input to RL policies. Updated the function docstring